src/settings/Membership/Membership.test.js

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import { Membership } from './Membership';
+
+const renderMembershipSettings = () => render(
+  <Membership />,
+  { wrapper: MemoryRouter },
+);
+
+describe('Membership', () => {
+  it('should display controlled vocabulary setting', async () => {
```

Comment:
Why is it async?

Reply:
it was just used from the test boilerplate, so function expression as asynchronous may have remained. I'll remove the keyword, but that will not change anything, to be honest)